commit a23954ec17c17259f75e0f4e647975bbf4ede058
Author: Dana Jansens <danakj@orodu.net>
Date:   Thu Sep 29 12:52:07 2011 -0400

    bug #5253 kde4 is out now, use kde4 menus by default.
    
    Startup scripts should export "XDG_MENU_PREFIX=kde-4-" but they export
    "XDG_MENU_PREFIX=kde-".

diff --git a/data/xsession/openbox-kde-session.in b/data/xsession/openbox-kde-session.in
index 14e3504..3572279 100644
--- a/data/xsession/openbox-kde-session.in
+++ b/data/xsession/openbox-kde-session.in
@@ -8,7 +8,7 @@ if test -n "$1"; then
 fi
 
 # Set the prefix for the menu layout to use
-export XDG_MENU_PREFIX="kde-"
+export XDG_MENU_PREFIX="kde-4-"
 
 # Clean up after GDM
 xprop -root -remove _NET_NUMBER_OF_DESKTOPS \
